cmd/proxy/config/swagger/openapi.json

```diff
@@ -2081,15 +2081,15 @@
         }
       }
     },
-    "/transaction/{txhash}": {
+    "/transaction/{txHash}": {
```

Comment:
there is another endpoint "/transaction/{hash}" down below, i think it's duplicated and it can be removed

Reply:
right, removed that one